packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/internal_bulk_resolve.ts

```diff
@@ -271,92 +254,18 @@ export async function internalBulkResolve<T>(
     { refresh: false }
   ).catch(() => {}); // if the call fails for some reason, intentionally swallow the error
 
-  const redacted = await authorizeAuditAndRedact(resolvedObjects, securityExtension, namespace);
-  return { resolved_objects: redacted };
-}
-
-/**
- * Checks authorization, writes audit events, and redacts namespaces from the bulkResolve response. In other SavedObjectsRepository
- * functions we do this before decrypting attributes. However, because of the bulkResolve logic involved in deciding between the exact match
- * or alias match, it's cleaner to do authorization, auditing, and redaction all afterwards.
- */
-async function authorizeAuditAndRedact<T>(
```

Comment:
All authz logic is moved to the security extension'sauthorizeAndRedactInternalBulkResolve method.